src/status_im2/contexts/quo_preview/wallet/account_origin.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im2.contexts.quo-preview.wallet.account-origin
```

Comment:
Quo preview can be simplified by using improvements added in [this PR](https://github.com/status-im/status-mobile/pull/16996). That will make the file shorter and less boilerplate :)

Reply:
We can go further @mmilad75. See comment https://github.com/status-im/status-mobile/pull/17147#discussion_r1310643313

Reply:
Done